src/Graphics/OpenGLContext/opengl_UnbufferedDrawer.h

```diff
@@ -19,9 +20,25 @@ namespace opengl {
 		void drawLine(f32 _width, SPVertex * _vertices) override;
 
 	private:
-		bool _updateAttribPointer(u32 _index, const void * _ptr);
+		bool _updateAttribPointer(u32 _type, u32 _index, const void * _ptr);
+		void _updateBuffer(u32 _type, u32 _length, const void * _data);
 
+		CachedBindBuffer * m_bindBuffer;
```

Comment:
We probably need two VBO implementation classes: one for buffer_storage and another for glMapBufferRange